torch/_dynamo/utils.py

```diff
@@ -927,6 +927,10 @@ def tuple_iterator_getitem(it, index):
     return obj[start + index]
 
 
+def iter_next(it):
+    return next(it)
```

Comment:
rm?

Reply:
no, this is needed to ensure that next doesn't shadow a user var named next when replaying mutations. We import the fully qualified name